crates/cargo-contract/src/cmd/extrinsics/integration_tests.rs

```diff
@@ -110,7 +105,6 @@ impl ContractsNodeProcess {
     }
 
     fn kill(&mut self) {
-        tracing::debug!("Killing contracts node process {}", self.proc.id());
```

Comment:
As this one is in a helper function and not directly in the test, let's bring it back.